keyboards/contra/keymaps/martin/config.h

```diff
@@ -0,0 +1,42 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the first 4 lines, eg, these lines: 
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "config_common.h"
```
with just `#pragma one`, and remove the endif at the end?